tests/test_sagemaker.py

```diff
@@ -157,6 +159,82 @@ def test_delete_notebook_instance(self):
         self.assertTrue(notebook['NotebookInstanceStatus'], 'Deleting')
 
 
+class TestModelInstance(BaseTest):
+    def test_list_model(self):
+        session_factory = self.replay_flight_data(
+            'test_sagemaker_model')
+        p = self.load_policy({
+            'name': 'list-sagemaker-model',
+            'resource': 'sagemaker-model'
+        }, session_factory=session_factory)
+        resources = p.run()
+        self.assertGreaterEqual(len(resources), 1)
+
+    def test_delete_model(self):
+        session_factory = self.replay_flight_data(
+            'test_sagemaker_delete_model')
+        p = self.load_policy({
+            'name': 'delete-invalid-sagemaker-model',
+            'resource': 'sagemaker-model',
+            'filters': [
+                {'tag:DeleteMe': 'present'}],
+            'actions': [{'type': 'delete'}]}, session_factory=session_factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+        client = session_factory().client('sagemaker')
+        try:
+            model = client.describe_model(
+                ModelName=resources[0]['ModelName'])
+        except b_exc.ClientError as e:
+            if e.response['Error']['Code'] != 'ValidationException':
+                self.fail('Bad Error:' + e.response['Error']['Code'])
+            else:
+                self.assertTrue(e.response['Error']['Code'], 'ValidationException')
```

Comment:
assertEqual? this seems odd that its passing,  and odd to check given the if condition.

Reply:
per pydoc, the signature here is unittest.TestCase.assertTrue = assertTrue(self, expr, msg=None) so probably not the intended outcome.